### PR TITLE
dash/unique-cell-id-warning

### PR DESCRIPTION
**Pull request by sebastianbochan**
Touch #20661, added warning when ids is duplicated.

### CODE REVIEW

**Review by jomar-honsi — APPROVED** [review]

File: ts/Dashboards/Actions/Bindings.ts [hunk 1]
```diff
@@ -76,6 +76,13 @@ namespace Bindings {
         let guiElement;
 
         if (typeof idOrElement === 'string') {
+            if (document.querySelectorAll('#' + idOrElement).length > 1) {
+                console.warn(
+                    'A few cells have the same id: `' + idOrElement +
```
[the remark this answers is not in the document]
Comment (karolkolodziej) [reply ts/Dashboards/Actions/Bindings.ts:81]: I disagree, with this implementation, the components are going to be rendered. 

I would phrase it like this:
`Multiple cells have identical ID ("' + idOrElement + '"), potentially leading to unexpected behaviour. Ensure that each cell has a unique ID on the page.`

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
- [x] See linter errors

File: ts/Dashboards/Actions/Bindings.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (stitot) [reply ts/Dashboards/Actions/Bindings.ts:81]: According to @sebastianbochan not all components will be rendered if there are identical ids in the same dashboard. All will be rendered only if identical ids across multiple dashboards. Anyways, your proposal is more general and catches any unidentified edge cases, so I like it better! Thanks!

[the remark this answers is not in the document]
Comment (sebastianbochan) [reply ts/Dashboards/Actions/Bindings.ts:81]: I agree, so we have a solution! :) 

Thank you for suggestions!

**Review by karolkolodziej — APPROVED** [review]

**Review by Vazonik — APPROVED** [review]

**Pull request merged**